spec/main.tex

```diff
@@ -886,13 +887,15 @@ \subsection{Provisioning Protocol}
 
       \item For $j \neq i$:
         \begin{itemize}
+          \item Assert $(N_j, s_j, t_j, \rho_j, u_j) \? \in
+            (\Z, \Z, \Z, \Bit^\kappa, \Bit^\kappa)$
           \item Assert $V_j =
-            H(\Encode{hash\_com}(\sid, n, j, N_j, s_j, t_j, \hat{\psi}_j, \rho_j, u_j))$.
+            H(\Encode{hash\_com}(\sid, j, N_j, s_j, t_j, \hat{\psi}_j, \rho_j, u_j))$.
           \item Assert $N_j$ is at least $8 \cdot \kappa - 1$ bits in length
```

Comment:
We only set a lower limit at the $N_j$, but shouldn't we set upper limit as well? I'm thinking that if adversary can set a large $N$, then protocol might significantly slow down.

Reply:
Well, in this case we literally follow the check mandated by the original paper. I agree with you, adding such a check makes sense

Reply:
Moved to #136